stories/molecules/components/networkGraph/tooltipExtractors.ts

```diff
@@ -0,0 +1,64 @@
+import type { Node } from 'ngraph.graph'
+import type { DeepReadonly } from 'superTypes'
+import type { TooltipExtractor } from 'ui/molecules/networkGraph/ecs/system/nodeTooltipSystem'
+import type { EvmosValidator, FamilyTreeObject } from './meshFactory'
+import femaleIcon from '../../../assets/female.svg'
+import maleIcon from '../../../assets/male.svg'
+
+export const familyTreeTooltipExtractor: TooltipExtractor<FamilyTreeObject> = (
+  node: DeepReadonly<Node<FamilyTreeObject>>
+) => {
+  const tooltip = document.createElement('div')
+  const ImgHTML = document.createElement('img')
+  const title = document.createElement('h3')
+  const datesWrapper = document.createElement('div')
+  const dateBirth = document.createElement('p')
+  const dateDeath = document.createElement('p')
+  const { name, gender, dateOfBirth, dateOfDeath, portrait }: FamilyTreeObject = node.data
+
+  const formatDate = (dateString: string): string => {
+    const [day, month, year]: string[] = dateString.split('/')
+    const date = new Date(`${year}-${month}-${day}`)
```

Comment:
You could also get the I18n language to display it in VF too (why not, it's just a detail).

Reply:
Great suggestion! But as you say it's a detail so if there is more work to be done on the story I might add it otherwise I prefer to go ahead and merge the PR as is.